Encapsulation Exercises/temperature-converter/src/Temperature.java

```diff
@@ -0,0 +1,26 @@
+public class Temperature {
+    private double celsius;
```

Comment:
with celsius as a field, how can you do fehrenheit to celsius conversion?
I suggest a neutral field first like `private double value;` then another field like `private String currentTemperatureUnit`
if currentTemperatureUnit is celsius then you can convert the value in fahrenheit and vice versa

Reply:
work on this